### PR TITLE
Fix environment install token

### PR DESCRIPTION
**Pull request by igorzi84**
1. Fix environment install token
2. groupdel -f flag not always available 
```
groupdel: invalid option -- 'f'
Usage: groupdel [options] GROUP

Options:
  -h, --help                    display this help message and exit
  -R, --root CHROOT_DIR         directory to chroot into
```

(Ubuntu 16.04.7)

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -665,7 +665,7 @@ function uninstall() {
             # remove user
             if getent passwd "${SYSTEM_USER}" > /dev/null; then
                 userdel -r -f "${SYSTEM_USER}"
-                groupdel -f "${SYSTEM_USER}"
+                groupdel "${SYSTEM_USER}"
```
Comment (swiatekm-sumo) [inline scripts/install.sh:668]: The `-f` was there so the command doesn't error if the group doesn't exist. If we want to get rid of it, I think we should do:
```suggestion
                groupdel "${SYSTEM_USER}" 2>/dev/null || true
```

Also gets rid of the annoying error message.

File: examples/chef/sumologic-otel-collector/resources/default.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@
   install_command = get_install_script_command(new_resource)
   execute INSTALL_SCRIPT_PATH do
     command install_command
-    environment ({"SUMOLOGIC_INSTALL_TOKEN" => resource.install_token})
+    environment ({"SUMOLOGIC_INSTALL_TOKEN" => new_resource.install_token})
```
Comment (sumo-drosiek) [inline examples/chef/sumologic-otel-collector/resources/default.rb:30]: We renamed install token to installation token, so the final change should be:

```suggestion
    environment ({"SUMOLOGIC_INSTALLATION_TOKEN" => new_resource.installation_token})
```

**Comment by sumo-drosiek** [comment]
@igorzi84 Please rebase/merge main and we should be good to go

**Comment by swiatekm-sumo** [comment]
> > @igorzi84 Please rebase/merge main and we should be good to go
> 
> Looks like I have some issues due to (I didnt change any of them)
> 
> ```
> Changes not staged for commit:
>   (use "git add <file>..." to update what will be committed)
>   (use "git restore <file>..." to discard changes in working directory)
>         modified:   docs/Comparison.md
>         modified:   docs/Configuration.md
>         modified:   docs/Installation.md
>         modified:   docs/Migration.md
>         modified:   docs/Performance.md
>         modified:   docs/Troubleshooting.md
>         modified:   docs/Upgrading.md
> ```
> 
> is there some workflow on those?

I'm guessing this is on MacOS? Unfortunately APFS is case-insensitive by default, and we have both lower and upper-case versions of those files. It's possible to have git handle this correctly, but it's probably easier if you just wait for https://github.com/SumoLogic/sumologic-otel-collector/pull/973 and rebase afterwards.

**Comment by sumo-drosiek** [comment]
quixkfix:
```
git update-index --assume-unchanged docs/{Comparison,Configuration,Installation,Migration,Performance,Troubleshooting,Upgrading}.md
```

**Comment by igorzi84** [comment]
> quickfix:
> 
> ```
> git update-index --assume-unchanged docs/{Comparison,Configuration,Installation,Migration,Performance,Troubleshooting,Upgrading}.md
> ```

Now its

```
 ✘  ~/code/sumologic-otel-collector   main  git status              
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
 ~/code/sumologic-otel-collector   main  git rebase upstream/main
error: Your local changes to the following files would be overwritten by checkout:
        docs/Comparison.md
        docs/Configuration.md
        docs/Installation.md
        docs/Migration.md
        docs/Performance.md
        docs/Troubleshooting.md
        docs/Upgrading.md
Please commit your changes or stash them before you switch branches.
Aborting
error: could not detach HEAD
```

**Comment by sumo-drosiek** [comment]
This is so bad. Seems that removing symbolic links crashes the `rebase`.
I would try to reset hard to `origin/main` and cherry-pick the changes, or use `Resolve conflicts` on github UI

**Comment by sumo-drosiek** [comment]
You can also try this command:

```
git update-index --no-assume-unchanged 
```

**Review by sumo-drosiek — APPROVED** [review]

**Comment by sumo-drosiek** [comment]
@igorzi84 Thank you for the contribution!

**Pull request merged**